charts/pokt-relay-meter/values.yaml

```diff
@@ -0,0 +1,144 @@
+# Default values for pokt-relay-meter
+# This is a YAML-formatted file
+# Declare variables to be passed into your templates
+
+replicaCount: 1
+
+collector:
+  image:
+    repository: 059424750518.dkr.ecr.us-west-2.amazonaws.com/relay-meter-collector
+    pullPolicy: IfNotPresent
+    tag: "latest"
```

Comment:
I would leave this blank, and then set the appVersion in Charts.yaml to match the semver tag